tests/load/load_test_all_read_setup_endpoints.sh

```
@@ -0,0 +1,543 @@
+    local method=$2
+    shift 2
+    local params="$*"
+    
+    total_tests=$((total_tests + 1))
+    
+    print_color "$CYAN" "Testing: $method"
+    printf "  Parameters: [%s]\n" "$params"
+    printf "  Status: "
+    
+    # Escape params for JSON
+    local escaped_params=$(echo "$params" | sed 's/\\/\\\\/g' | sed 's/"/\\"/g')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)